docs/obis-observer/commands/GetShortName.md

```diff
@@ -0,0 +1,99 @@
+# GetShortName
+
+Request/response the short name list of the specific OBIS code.
+
+
+## Request
+
+### Format
+
+| Size  | Type   | Field                     |
+| ----- | ------ | ------------------------- |
+| `1`   | `byte` | command id = `0x01`       |
+| `1`   | `byte` | [request id](#request-id) |
+| `3-7` | `byte` | [obis](#obis)             |
+
+### Parameters
+
+#### **request id**
+
+It's a unique number for pair device and GetShortName correction request.
+Then increment this value for each following request.
```

Comment:
It should be incremented for each request.

Reply:
description for request id changed